eng/Version.Details.xml

```diff
@@ -293,34 +293,38 @@
       <Uri>https://github.com/dotnet/llvm-project</Uri>
       <Sha>0a471ebbe6a4a9c095169dd531b825a4370f57f3</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="8.0.0-rc.1.23406.6">
+    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="8.0.10">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>edbd5c769a19798b6955050baccf99e6797d3208</Sha>
+      <Sha>b5f53494d6541a1ef9b7d0e13214d5e5e5cec594</Sha>
     </Dependency>
     <Dependency Name="runtime.native.System.IO.Ports" Version="8.0.0-rc.1.23406.6">
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>edbd5c769a19798b6955050baccf99e6797d3208</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.ILAsm" Version="8.0.0-rc.1.23406.6">
+    <Dependency Name="Microsoft.NETCore.ILAsm" Version="8.0.10-servicing.24466.9">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>edbd5c769a19798b6955050baccf99e6797d3208</Sha>
+      <Sha>b5f53494d6541a1ef9b7d0e13214d5e5e5cec594</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NET.Sdk.IL" Version="8.0.0-rc.1.23406.6">
+    <Dependency Name="Microsoft.NET.Sdk.IL" Version="8.0.10-servicing.24466.9">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>edbd5c769a19798b6955050baccf99e6797d3208</Sha>
+      <Sha>b5f53494d6541a1ef9b7d0e13214d5e5e5cec594</Sha>
     </Dependency>
-    <Dependency Name="System.Text.Json" Version="8.0.0-rc.1.23406.6">
+    <Dependency Name="System.Text.Json" Version="8.0.5">
```

Comment:
I double-checked that STJ 8.0.6 was not a security update. So bumping only to 8.0.5 is good.